config_dev.toml.example

```diff
@@ -0,0 +1,139 @@
+# Linkedevents environment configuration
+# This file defines a set of (environment) variables that configure most
+# of the functionality of linkedevents. In order for linkedevents to read
+# this file, rename it to `config_dev.toml`. As the name implies, this
+# file is supposed to be used only in development. For production use
+# we recommend setting the environment variables using the facilities
+# of your runtime environment.
+# Linkedevents reads this file by itself. However, it can also be
+# used in conjunction with the included docker-compose.yml. Then
+# you don't need to inject the file into the container. Instead
+# Docker defines environment variables that linkedevents will read.
+# Note that although this file has the extension .toml, it is actually
+# read by a parser only capable of reading key=value pairs + comments.
+# We use the .toml extension to indicate that it is readable by TOML
+# parser.
+# Following are the settings and their explanations, with useful values
+# for development:
+
+# Whether to run Django in debug mode
+# Django setting: DEBUG https://docs.djangoproject.com/en/2.2/ref/settings/#debug
+DEBUG=True
+
+# Namespace ID used for events added through the API. All events added
+# through api will have their "data_source" field set to this value and
+# their identifier will be prefixed by this value.
+SYSTEM_DATA_SOURCE_ID=ledev
+
+# Languages that this Linkedevents instance will support. This affects
+# several things and cannot be changed nilly-willy. Most importantly,
+# you will need to run `manage.py sync_translation_fields` after
+# changing this. You will also need to rebuild your elasticsearch
+# indexes: `manage.py rebuild_indexes`, but only if you are actually
+# running elasticsearch, ie. specifying ELASTICSEARCH_URL
+# Django setting: LANGUAGES https://docs.djangoproject.com/en/2.2/ref/settings/#languages
+LANGUAGES=fi,en
```

Comment:
The default languages should be the same as those in `settings.py`, as existing migrations are done assuming that. Then, `sync_translation_fields` only updates munigeo tables and everything else works with no changes.

Therefore, the languages should be

```

    ('fi', 'Finnish'),
    ('sv', 'Swedish'),
    ('en', 'English'),
    ('zh-hans', 'Simplified Chinese'),
    ('ru', 'Russian'),
    ('ar', 'Arabic'),
)
```

with the names included. How are those names generated here?

Reply:
Again, I'll comment this out too. It is a bit of shame, that the migrations require a certain set of languages though.
See here about the language names: https://github.com/City-of-Helsinki/linkedevents/blob/28521539e285d15ca354537cf0c79a1ee2b1a594/linkedevents/settings.py#L159-L167

Reply:
Oh. Nice you can just read it here. 😁

Reply:
There really was no point in repeating the translations done by Django developers.

Reply:
Well, the migrations probably strictly don't require a certain set, `sync_translation_fields` might be able to remove as well as add languages. Haven't tested that. However, perhaps best use the default set?